xcp_d/workflow/postprocessing.py

```diff
@@ -24,18 +24,12 @@
 def init_post_process_wf(
         mem_gb,
         TR,
-        head_radius,
         lower_bpf,
         upper_bpf,
         bpf_order,
         smoothing,
         bold_file,
         params,
-        motion_filter_type,
-        motion_filter_order,
-        band_stop_max,
-        band_stop_min,
-        initial_volumes_to_drop,
```

Comment:
Some of these need to be removed from the example call and the parameters list in the docstring.